src/Kiosk.Api/Controllers/NewsController.cs

```diff
@@ -57,13 +57,16 @@ public NewsController(ILogger logger, INewsService newsService)
         CancellationToken cancellationToken,
         [FromQuery, Required]
         Language language,
-        [FromQuery] Source? source=null)
+        [FromQuery] Source? source=null,
+        [FromQuery(Name = "page")] int page = 1,
+        [FromQuery(Name = "itemsPerPage")] int itemsPerPage = 30
```

Comment:
And First of all why we don't create a DTO for pagination? then it's just 
`[FromQuery] pagination` and we can declare a default values there